src/widget/wtracktableview.cpp

```diff
@@ -1061,6 +1099,7 @@ void WTrackTableView::contextMenuEvent(QContextMenuEvent* event) {
         m_pMenu->addAction(m_pPurgeAct);
     }
     m_pMenu->addAction(m_pFileBrowserAct);
+    m_pMenu->addAction(m_pFileRemoveFromDiskAct);
```

Comment:
I suppose this action is not used very often and at least not while performing. It doesn't deserve enlarging and cluttering the top level layer of the context menu.

Reply:
What about a `Hide / Delete` submenu?
Could this hold items that distinguish between Hide, Delete but keep in history, Delete & Purge etc. or is it better to decide that per file in the dialog window?

Reply:
I'm not too worried about cluttering context menu...  How often do you really right-click while performing?  I do most of my file selection with a hardware controller and never touch a mouse anyways.  I would not want another dialog box for each file...  If I selected a range, then hit purge, then purge all I selected...